boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts

```diff
@@ -204,10 +204,9 @@ i2s1: &flexcomm3 {
 };
 
 &flexspi {
-	reg = <0x50134000 0x4000>, <0x18000000 DT_SIZE_M(64)>;
 	mx25um51345g: mx25um51345g@2 {
 		compatible = "nxp,imx-flexspi-mx25um51345g";
-		size = <536870912>;
+		size = <DT_SIZE_M(64)>;
```

Comment:
@saurabh-nxp Any reason we included the size to be `536870912`

Reply:
From @saurabh-nxp `536870912` is `512M`. From the `mx25um51345g` this should be 64MB, so this change looks correct.